### PR TITLE
Move imports to the top

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: python/HeaderService/hutils.py [hunk 1]
```diff
@@ -192,13 +194,11 @@ def get_date_utc(timeStamp=None, format='isot'):
...
 
-    from astropy.time import Time
-    from datetime import datetime
     if timeStamp is None:
-        utc_time = (datetime.utcnow()).isoformat()
+        utc_time = (datetime.datetime.utcnow()).isoformat()
     else:
-        utc_time = datetime.utcfromtimestamp(timeStamp).isoformat()
-    t = Time(utc_time, format=format, scale='utc')
+        utc_time = datetime.datetime.utcfromtimestamp(timeStamp).isoformat()
```
Comment (timj) [inline python/HeaderService/hutils.py]: You can avoid the unnecessary stringification here by going directly to Astropy Time:
```
t = astropy.time.Time(timeStamp, format="unix")
```

File: python/HeaderService/hutils.py [hunk 2]
```diff
@@ -192,13 +194,11 @@ def get_date_utc(timeStamp=None, format='isot'):
 
     """
 
-    from astropy.time import Time
-    from datetime import datetime
     if timeStamp is None:
-        utc_time = (datetime.utcnow()).isoformat()
+        utc_time = (datetime.datetime.utcnow()).isoformat()
```
Comment (timj) [inline python/HeaderService/hutils.py]: Avoid the extra stringification by doing:
```
t = astropy.time.Time.now()
```

File: python/HeaderService/hutils.py [hunk 3]
```diff
@@ -140,7 +143,6 @@ def md5Checksum(filePath, blocksize=1024*512):
 
 
 def get_obsnite(date=None, thresh_hour=14, format='{year}{month:02d}{day:02d}'):
```
Comment (timj) [inline python/HeaderService/hutils.py:159]: Is this function used any more now that DAYOBS is being extracted from OBSID?

File: python/HeaderService/hutils.py [hunk 4]
```diff
@@ -120,7 +124,6 @@ def get_values(header):
 
 
 def elapsed_time(t1, verb=False):
```
Comment (timj) [inline python/HeaderService/hutils.py:126]: Methods need doc strings.

File: python/HeaderService/hutils.py [hunk 5]
```diff
@@ -33,6 +33,10 @@
 import itertools
 import copy
 from .camera_coords import CCDGeom
+import astropy
```
Comment (timj) [inline python/HeaderService/hutils.py]: The usual convention in Astropy is to do `from astropy.time import Time`

**Review by timj — APPROVED** [review]
Looks ok. I think the astropy time code should be cleaned up to stop generating an intermediary ISO format string that we don't need.

File: python/HeaderService/hutils.py (hunk 1 above)
Comment (menanteau) [reply python/HeaderService/hutils.py]: Thanks for the suggestion, it's been implemented.

File: python/HeaderService/hutils.py (hunk 2 above)
Comment (menanteau) [reply python/HeaderService/hutils.py]: Done too.

File: python/HeaderService/hutils.py (hunk 3 above)
Comment (menanteau) [reply python/HeaderService/hutils.py:159]: Not anymore, your comment implies that it is okay to get rid of it. Therefore that what I'll do.

File: python/HeaderService/hutils.py (hunk 5 above)
Comment (menanteau) [reply python/HeaderService/hutils.py]: Updated to `from astropy.time import Time` at the top of the file.

File: python/HeaderService/hutils.py (hunk 4 above)
Comment (menanteau) [reply python/HeaderService/hutils.py:126]: Added doc string.

**Pull request merged**